README.md

```diff
@@ -32,41 +32,96 @@ Configure a webhook for your Github repository following the instructions [here]
-8. Get your Jenkins API token by logging in to https://manhattan.ci.chef.co, clicking your name on the top right, and clicking Configure. Then click "Show API Token" and copy the values for the next step.
-9. Create `lita-versioner/lita_config.rb` with this text:
+### First Time Setup
+
+1. Install [Docker Toolbox](https://www.docker.com/products/docker-toolbox):
+   - `brew cask install dockertoolbox`
+   - `docker-machine create default --driver virtualbox`
+   - `eval $(docker-machine env default)` <-- you might want that in your .profile
+2. Copy `lita_config.rb.example` to `lita_config.rb` with this text:
    ```ruby
    Lita.configure do |config|
-     config.handlers.versioner.jenkins_username = "YOUR USERNAME"
-     config.handlers.versioner.jenkins_api_token = "YOUR TOKEN"
+     config.handlers.versioner.jenkins_username = "YOUR JENKINS USERNAME"
+     config.handlers.versioner.jenkins_api_token = "YOUR TOKEN FROM STEP 4"
      config.robot.adapter = :shell
      config.robot.log_level = :debug
    end
    ```
-10. Run lita within the lita development environment:
-  * `bundle exec lita` from `workspace/lita-versioner` on your lita dev VM.
-  * You should see this:
-    ```
-    Type "exit" or "quit" to end the session.
-    Lita >
-    ```
-  * From here you can interact with the bot just like on Slack.
+
+This will use your personal configuration.
+
+### Running Lita
+
+To run lita, do:
+
+```
+docker/run
+```
+
+You should see this towards the end:
+
+```
+Type "exit" or "quit" to end the session.
+Lita >
+```
+
+From here you can interact with the bot just like on Slack. Type `@lita help` for a list of commands.
+
+### Testing Against Slack
+
+To test that the bot can communicate on Slack, you need to set up the bot and join it to the channel:
+
+1. Set up the bot on Slack.
+   - Go to the [new Lita bot page](https://chefio.slack.com/apps/new/A0F7XDUJH-lita) on Slack.
+   - Create a username, like `<YOUR USERNAME>-testbot`.
+   - Copy the Slack token.
+2. Set up a channel for testing and invite your bot to it.
+   - /join #<YOUR USERNAME>-testbot
+   - /invite @<YOUR USERNAME>-testbot
+3. Configure the bot locally.
+   - Bring over the lita_config.rb.example slack configuration, inserting the token from step 1 and choosing a channel:
+     ```
+     config.adapters.slack.token = "<MY API TOKEN>"
+     config.adapters.slack.link_names = true
+     config.adapters.slack.parse = "full"
+     config.adapters.slack.unfurl_links = false
+     config.adapters.slack.unfurl_media = false
+     config.default_inform_channel = "#<MY TEST CHANNEL>"
+     ```
+
+Now when you start the bot, it should join!
+
+### Testing Against Jenkins
+
+To test against Jenkins, you just need to add the correct tokens to the config file.
+
+1. Get your Jenkins API token by logging in to https://manhattan.ci.chef.co, clicking your name on the top right, and clicking Configure. Then click "Show API Token" and copy the values for the next step.
```

Comment:
I rearranged it so jenkins isn't part of the initial setup. No more reference to step 4 :)